qlib/workflow/__init__.py

```diff
@@ -16,8 +16,11 @@ class QlibRecorder:
     def __init__(self, exp_manager):
         self.exp_manager = exp_manager
 
+    def __repr__(self):
+        return "{name}(manager={manager})".format(name=self.__class__.__name__, manager=self.exp_manager)
+
     @contextmanager
-    def start(self, experiment_name=None, recorder_name=None):
+    def start(self, experiment_name=None, recorder_name=None, uri=None):
```

Comment:
Thanks for pointing this out! 

Apparently, adding `uri` argument in this method will not change the one defined in the `config` file. Therefore, the next time when user call this function in the same experiment, they have to also specify this argument with the same value. Otherwise, inconsistent `uri` may occur.

Could you please also update the related `doc` to specify this situation? Thanks~

Reply:
Sure, added :)